g11n-ws/vip-manager-i18n/src/main/java/com/vmware/vip/core/conf/WebConfiguration.java

```diff
@@ -91,6 +91,9 @@ public class WebConfiguration implements WebMvcConfigurer {
 	
 	@Value("${config.client.requestIds:}")
 	private String requestIdsStr; 
+	
+	@Value("${source.request.max-size:10485760}")
```

Comment:
Is this '10485760' duplicated with the one in the class 'CollectSourceReqBodyInterceptor'? can remove it?

Reply:
done